nest/src/auth/auth.service.ts

```diff
@@ -5,7 +5,8 @@ import {
 } from '@nestjs/common';
 import { JwtService } from '@nestjs/jwt';
 import { UsersService } from '../users/users.service';
-import { CreateUserDto } from '../users/dto/create-user.dto';
+import { JwtUserDto } from '../users/dto/jwt-user.dto';
+import { Intra42UserDto } from '../users/dto/intra42-user.dto';
```

Comment:
Here since we don't send over and receive such payload type, then it's really a DTO as far as I understood. I guess there are rather interfaces